test/Interop/Cxx/class/synthesized-initializers.swift

```diff
@@ -0,0 +1,46 @@
+// RUN: %target-swift-frontend -I %S/Inputs -enable-cxx-interop -emit-silgen %s | %FileCheck %s
+
+import LoadableTypes
+
+// CHECK-LABEL: sil shared [transparent] [serializable] [ossa] @$sSo11EmptyStructVABycfC
```

Comment:
Could you add the parameter list and the return type to the CHECK line?

Reply:
Sorry, did you forget to push a commit? I was expecting a colon and a type signature after `@$sSo11EmptyStructVABycfC`.

Reply:
Oh, I see. My bad. I thought you were talking about the BB args.

Reply:
Done.